rhombus/src/internal/routes/scoreboard.rs

```diff
@@ -37,12 +56,20 @@ pub async fn route_scoreboard_division(
     state: State<RouterState>,
     Extension(user): Extension<MaybeUser>,
     Extension(page): Extension<PageMeta>,
-    Path(division_id): Path<i64>,
+    Path(division_id): Path<String>,
     params: Query<PageParams>,
-    req: Request<Body>,
+    uri: Uri,
 ) -> impl IntoResponse {
     let page_num = params.page.unwrap_or(1).saturating_sub(1);
 
+    let Ok(division_id) = division_id
+        .strip_suffix(".json")
+        .unwrap_or(&division_id)
+        .parse()
+    else {
+        return Err::<(), &str>("Failed to parse integer").into_response();
```

Comment:
Maybe should be a little nicer error message, like `Failed to parse division id as integer`. Although there is little error handling in this route anyways so not a big deal.